task7/second/client/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"os"
+)
+
+func main() {
+	conn, err := net.Dial("tcp", "127.0.0.1:8080")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer conn.Close()
+
+	for {
+		reader := bufio.NewReader(os.Stdin)
+		text, _ := reader.ReadString('\n') // read msg from stdin
+		fmt.Fprintf(conn, text+"\n")       // send msg
+		msg := make([]byte, 100)           // buffer where we will read msg from server
```

Comment:
It's ok for current task, but keep in mind response can be more than a 100 bytes

Reply:
Yes, I know, this only for test)